file/file.go

```diff
@@ -212,6 +216,62 @@ func NewWriter(pth string, opt *Options) (w Writer, err error) {
 	return w, err
 }
 
+type Iterator struct {
+	err     error
+	isValid bool
+	reader  Reader
+}
+
+func NewIterator(path string, opts *Options) *Iterator {
+	r, err := NewReader(path, opts)
+	return &Iterator{
+		err:     err,
+		isValid: err == nil,
+		reader:  r,
+	}
+}
+
+func (i *Iterator) Lines() iter.Seq[[]byte] {
```

Comment:
Could we add some comments on how this Lines() function works. I'm sure I would hover over it to see how to use it when I import task tools.